wp-content/themes/access/singular.php

```diff
@@ -25,15 +25,18 @@
 /**
  * Set Alerts
  */
-
-$alerts = Timber::get_posts(array(
-  'post_type' => 'alert',
-  'posts_per_page' => -1
-));
-
-$context['alerts'] = array_filter($alerts, function($p) {
-  return in_array('pages', array_values($p->custom['location']));
-});
+if (get_field('alert')) {
+  $context['alerts'] = get_field('alert');
+} else {
+  $alerts = Timber::get_posts(array(
+    'post_type' => 'alert',
+    'posts_per_page' => -1
+  ));
+
+  $context['alerts'] = array_filter($alerts, function($p) {
+    return in_array('pages', array_values($p->custom['location']));
+  });
+}
 
 /**
  * Render Template
```

Comment:
I think you need to update your branch... These changes should already be there.